src/librbd/io/CopyupRequest.cc

```diff
@@ -264,7 +256,11 @@ bool CopyupRequest<I>::should_complete(int r)
 
   case STATE_COPYUP:
     // invoked via a finisher in librados, so thread safe
```

Comment:
@dillaman I assume this comment is not relevant now?

Reply:
Yanked it